src/Umbraco.Tests/Testing/UmbracoTestBase.cs

```diff
@@ -308,9 +308,11 @@ protected virtual void ComposeWeb()
 
             // web
             Builder.Services.AddUnique(_ => Umbraco.Web.Composing.Current.UmbracoContextAccessor);
+            Builder.Services.AddUnique<IBackOfficeSecurityAccessor>(_ => new HybridBackofficeSecurityAccessor(AppCaches.NoCache.RequestCache));
```

Comment:
Why are these two lines registering factory methods instead of just registering instances?

Reply:
resolved #7e9a13a